src/pages/industries/financial-services.jsx

```diff
@@ -0,0 +1,230 @@
+  ],
+
+  withImage: true,
+  imageSrc: FinanceImage1,
+  imageAlt: 'Cilium continuous compliance image',
+};
+
+const financialUsecases = [
+  {
+    icon: EncryptionIcon,
+    title: 'Transparent Encryption ',
+    url: '/use-cases/transparent-encryption',
+    description:
+      'Elevate compliance and lower risk with Cilium transparent encryption. With just one switch, no application changes, service meshes  or additional proxies',
+  },
+  {
+    icon: NetworkPolicyIcon,
+    title: 'Network Policy ',
+    url: '/use-cases/network-policy',
+    description:
+      'Maintain identity based policies effectively at scale with Cilium’s advanced network polices ',
+  },
+  {
+    icon: EgressGatewayIcon,
+    title: 'Egress Gateway',
+    url: '/use-cases/egress-gateway',
+    description:
+      'Present a group of cloud native workloads from a stable IP address to integrate with traditional firewalls.',
+  },
+];
+
+const financialTalks = [
+  {
+    title: "eBPF, a road to invisible network: S&P Global's Network Transformation Journey",
+    description:
+      'Guru Ramamoorthy talks on how S&P Global’s network engineers leveraged eBPF-based networking with Cilium to power their application.',
+    videoSrc: 'https://www.youtube.com/embed/6CZ_SSTqb4g',
+  },
+  {
+    title: 'Cilium in Practice: Building Data Sandboxes at Bloomberg',
+    videoSrc: 'https://www.youtube.com/embed/8fiYVyISyz4',
+    description:
+      'Bloomberg leverages Cilium to construct data sandboxes that restrict users from distributing data outside the sandbox',
+  },
+  {
+    title: 'Multi-cluster networking with Cilium at Form3',
+    description: 'This address the reasons Form3 choosed to build its platform with Cilium',
+    videoSrc: 'https://www.youtube.com/embed/vKgRf4OzTIE?si=w_-F3oQysthruVtA',
+  },
+];
+
+const financialResources = [
+  {
+    imageSrc: PostFinanceOfficeImage,
+    imageAlt: 'post finance office',
+    title: ' Post Finance picks Cilium for Cloud Native Networking',
+    url: 'https://www.youtube.com/watch?v=fjhKw49YcY4',
+    CTAtext: 'Watch The Case Study',
+    description:
+      'Cilium helped the post finance team build a scalable Kubernetes platform which meets the demanding requirements to run mission-critical banking software in production.',
+  },
+  {
+    imageSrc: CapitalOnefficeImage,
+    imageAlt: 'capital office building',
+    title: ' How Capital One used eBPF and Cilium to build a secure, maintainable PaaS',
+    url: 'https://www.youtube.com/watch?v=hwOpCKBaJ-w',
+    CTAtext: 'Watch The Talk',
+    description:
+      'Capital One built an internal PaaS called "Dragon" for its developers, based on Kubernetes. Dragon’s goal was to enable developers to ship code to production with little friction',
```

Comment:
can you make this about Cilium?

Reply:
should also be a . on the end

Reply:
Capital One leveraged Cilium to build a multi-tenant platform meeting all its requirements for security, maintainability, network visibility, and scale.